src/Boot/src/BootloadManager.php

```diff
@@ -22,15 +15,13 @@
  */
 final class BootloadManager implements Container\SingletonInterface
 {
-    /* @var Container @internal */
-    protected $container;
-
     /** @var array<class-string> */
-    private $classes = [];
+    private array $classes = [];
 
-    public function __construct(Container $container)
-    {
-        $this->container = $container;
+    public function __construct(
+        /* @internal */
+        protected Container $container
```

Comment:
readonly?

Reply:
Removed AbstractKernel::init, `protected Container $container` changed to `private readonly Container $container`. This class is final and protected doesn't make sense.